### PR TITLE
sync darts

### LINKED ISSUE #853: Sync darts
- [x] Run `bin/configlet sync -u -e darts --yes --docs --filepaths --metadata --tests include` (updates the Markdown files and maybe `tests.toml`)
  - docs: instructions unsynced: darts
  - metadata: unsynced: darts
- [x] Drop strict types comments from test file and example code (these are useless)
- [x] Add and sync test meta data to tests (`uuid` / `@testdox` in DocBlocks)
- [x] Decide on adding / adjusting / ordering test cases to match current problem specs
  - darts: missing 13 test cases

Do not redesign the student's interface or add test cases that would invalidate existing community solutions. These are extra tasks, which should be discussed in advance.

### PR DESCRIPTION
**Pull request by tomasnorre**
Resolves #853

### CODE REVIEW

**Comment by mk-mxp** [comment]
Discussion about a re-make in #853

File: exercises/practice/darts/.meta/example.php [hunk 1]
```diff
@@ -1,52 +1,20 @@
...
- * For more info review the Concept on strict type checking in the PHP track
- * <link>.
- *
- * To disable strict typing, comment out the directive below.
- */
-
 declare(strict_types=1);
 
-class Darts
+function calculateScore(float $xAxis, float $yAxis): int
```
Comment (mk-mxp) [inline exercises/practice/darts/.meta/example.php:5]: Canonical data uses `"property": "score",`, so we should use `score` as the functions name:

```suggestion
function score(float $xAxis, float $yAxis): int
```

File: exercises/practice/darts/Darts.php [hunk 2]
```diff
@@ -24,12 +24,7 @@
 
 declare(strict_types=1);
 
-class Darts
+function __construct(float $xAxis, float $yAxis)
```
Comment (mk-mxp) [inline exercises/practice/darts/Darts.php:27]:
```suggestion
function score(float $xAxis, float $yAxis): int
```

File: exercises/practice/darts/Darts.php [hunk 3]
```diff
@@ -24,12 +24,7 @@
...
 {
-    public int $score = 0;
-
-    public function __construct(float $xAxis, float $yAxis)
-    {
-        throw new BadFunctionCallException("Please implement the Darts class!");
-    }
+    throw new BadFunctionCallException("Please implement the Darts class!");
```
Comment (mk-mxp) [inline exercises/practice/darts/Darts.php:29]:
```suggestion
    throw new BadFunctionCallException("Please implement the __FUNCTION__ function!");
```

**Review by mk-mxp — CHANGES_REQUESTED** [review]
Thanks for taking care of this!

File: exercises/practice/darts/.meta/example.php (hunk 1 above)
Comment (tomasnorre) [reply exercises/practice/darts/.meta/example.php:5]: My bad, will correct that later.

**Review by mk-mxp — APPROVED** [review]
Thanks a lot!

**Pull request merged**